voluncheer/opportunityboard/management/commands/archive_opportunities.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+import pytz
+
+from opportunityboard.models import Opportunity
+
+
+class Command(BaseCommand):
+    help = "Archive opportunities that have ended"
+
```

Comment:
You could add an argument to run the command for a given day for testing purposes (see comment below). 

```
    def add_arguments(self, parser):
        parser.add_argument(
            "--datetime",
            default=datetime.now(pytz.utc),
            help="The date before which all opportunities are archived",
        )
```

You would then use` options["datetime"]` in place of `now `elsewhere in the command.

Reply:
This is an interesting idea! I will applied the changes. WYTD?